backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/service/CardVerifier.kt

```diff
@@ -1,24 +1,27 @@
 package app.ehrenamtskarte.backend.verification.service
 
+import app.ehrenamtskarte.backend.verification.database.CardEntity
 import app.ehrenamtskarte.backend.verification.database.repos.CardRepository
-import app.ehrenamtskarte.backend.verification.domain.Card
 import com.eatthepath.otp.TimeBasedOneTimePasswordGenerator
 import org.jetbrains.exposed.sql.transactions.transaction
 import java.time.Duration
 import java.time.Instant
+import java.time.LocalDateTime
 import javax.crypto.spec.SecretKeySpec
 
 val TIME_STEP: Duration = Duration.ofSeconds(30)
-const val TOTP_LENGTH = 10
+const val TOTP_LENGTH = 6
```

Comment:
We use 6 as 10 is out of range. Actually we could use any value in range here security wise.